pip/qsharp/telemetry_events.py

```diff
@@ -23,7 +23,10 @@ def get_shots_bucket(shots: int) -> int:
 
 
 # gets the order of magnitude for the number of qubits
-def get_qubits_bucket(qubits: int) -> int:
+def get_qubits_bucket(qubits: str) -> str:
```

Comment:
Why did this change from int to str, when most of the time it still returns an int?

Reply:
The RE data that gets returned is kind of a nonstandard format, and I wanted to be able to flag if that logicalQubits isn't where I expected it to be. I'll stringify the returns to match the type checking